tests/integration/targets/dcnm_vpc_pair/tests/dcnm/dcnm_vpc_pair_merge.yaml

```diff
@@ -286,6 +285,8 @@
     - name: Create VPC switch pair without state and deploy flag
```

Comment:
You added the `deploy` and `state` flag here but it looks like the purpose of this test was to intentionally leave the state and deploy flag out and pick up the default values for those properties?